Updated info to add PGN mainnet and testnet

sodew opened the pull request:
Corrected the addresses. Please let me know if this needs updating or if there is more information needed from PGN.

mmv08 commented on .history/src/assets/v1.3.0/compatibility_fallback_handler_20231102170232.json:
```diff
@@ -0,0 +1,478 @@
+{
```
What are those files? Please remove them

sodew replied:
These were removed

mmv08 commented on src/assets/v1.3.0/compatibility_fallback_handler.json:
```diff
@@ -123,6 +124,7 @@
     "54211": "0x017062a1dE2FE6b99BE3d9d37841FeD19F573804",
     "56288": "0x017062a1dE2FE6b99BE3d9d37841FeD19F573804",
     "57000": "0x017062a1dE2FE6b99BE3d9d37841FeD19F573804",
+    "58008": "0x1AC114C2099aFAf5261731655Dc6c306bFcd4Dbd",
```
this address doesn't match the expect address

mmv08 commented on src/assets/v1.3.0/sign_message_lib.json:
```diff
@@ -51,6 +51,7 @@
     "336": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "338": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
     "420": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
+    "424": "0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4",
```
This address doesn't match the expected one

sodew commented:
@mmv08 

Fixed PR so that .history files are removed, and mainnet contracts match expected addresses.

Additionally, removed testnet for now since some of the contracts aren't verified on PGNs end.

mmv08 commented:
> Additionally, removed testnet for now since some of the contracts aren't verified on PGNs end.

It is still present in one of the files. Also, it's okay with us if the contract isn't verified since the deployment is deterministic and depends on the hash of the bytecode. If one were to alter the code, it would result in a different address.
https://github.com/safe-global/safe-deployments/pull/315/files#diff-905275e3e6a99d588ff498e27b13efa9ee12d1ae314645f224cecca748ed98feR126

sodew commented:
@mmv08 Just added back testnet with expected contract addresses. 

Is there more we need to ensure is done on the PGN side to ensure this PR gets merged?
